Symfony/CS/Fixer/Contrib/MbStrFunctionsFixer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+class MbStrFunctionsFixer extends AbstractFixer
+{
+    /**
+     * @var array the list of the string-related function names and their mb_ equivalent
+     */
+    private static $functions = array(
```

Comment:
My PR is into `1.11`: is it ok for you if I leave this PR here to be merged on `1.11` and open another one for `master` using the linked abstract class?